package.json

```diff
@@ -425,6 +428,7 @@
     "eslint-plugin-cypress": "2.14.0",
     "eslint-plugin-import": "2.27.5",
     "eslint-plugin-jsx-a11y": "6.7.1",
+    "eslint-plugin-playwright": "^0.15.3",
```

Comment:
We have preferred to pin our versions for consistency and more explicit updates.

Reply:
Pinned :+1: